src/systemd/sd-gpt.h

```diff
@@ -29,6 +29,8 @@ _SD_BEGIN_DECLARATIONS;
 #define SD_GPT_ROOT_ARM64       SD_ID128_MAKE(b9,21,b0,45,1d,f0,41,c3,af,44,4c,6f,28,0d,3f,ae)
 #define SD_GPT_ROOT_IA64        SD_ID128_MAKE(99,3d,8d,3d,f8,0e,42,25,85,5a,9d,af,8e,d7,ea,97)
 #define SD_GPT_ROOT_LOONGARCH64 SD_ID128_MAKE(77,05,58,00,79,2c,4f,94,b3,9a,98,c9,1b,76,2b,b6)
+#define SD_GPT_ROOT_MIPS        SD_ID128_MAKE(e9,43,45,44,6e,2c,47,cc,ba,e2,12,d6,de,af,b4,4c)
```

Comment:
please also add _GPT_ARCH_SEXTET() entries for this to the  `gpt_partition_type_table[]` table in src/shared/gpt.c

Reply:
also add a table entry to man/systemd-gpt-auto-generator.xml

Reply:
Thanks, updated.